src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -40,6 +40,19 @@ public AnalyzeCommandBaseTests(ITestOutputHelper output)
             this.Output = output;
         }
 
+        [Fact]
+        public void AnalyzeCommandBase_RootContextIsDisposed()
```

Comment:
>AnalyzeCommandBase_RootContextIsDisposed [](http://example.com/codeflow?start=20&length=40)

Observe my test approach. First I add a focused test for both single and multithreaded analysis. This test, however, does not come close to testing all possible exit paths (this code returns in various places to handle exceptional paths, for example). So I will go inject this validation directly into our test commands as well.

This is a general strategy of mine when adding tests to legacy code: author directed tests, but look for opportunities to generalize coverage in the test infrastructure, via dependency injection, etc.